modules/ossm-member-roll.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-member-roll_{context}"]
+= Configure the {ProductName} member roll
```

Comment:
This section is out of order.  Should be after creating the control plane.

Reply:
I've moved this module to the end of the installing-ossm.adoc assembly. It appears after ossm-control-plane-deploy.adoc now.